kanopi/kanopi-components
# WP-CLI Site Reporting Commands

## Description
> As a developer, I need to retrieve common data from sites. For WordPress this includes Post, Taxonomy, and other related data, for instance ACF Flex Content field usage.

This work adds the infrastructure for common, reusable WP CLI commands to pull into any project. Any available commands are only made accessible through opt-in registration.

- Add the ACF flex content report CLI command

## Steps to Validate
1. Add the Composer library to a project
1. Ensure the auto-loader is registered
1. Following auto-loader registrations, add `Kanopi\Components\Commands\Registration::WPCLICommands();` to register all common commands
1. Use any of the commands, for instance `wp kanopi-flex-report-content audit ./ prefix- flex_content,area_list` where the comma-delimited list is the set of slugs to report, files are generated in the current directory with the prefix, `prefix-`
